data/json/items/magazines.json

```diff
@@ -71,5 +70,235 @@
         "alternatives" : [
             [ "9mm", [ "glockmag" ] ]
         ]
+    },{
+        "type" : "MAGAZINE",
+        "id" : "m1918mag",
+        "//" : "Compatable magazines made for reproductions exist, and the standard-capacity mags are fairly cheap compared to variants.",
+        "price" : 3600,
+        "name" : "BAR 20-round magazine",
+        "symbol" : "#",
+        "color" : "ltgray",
+        "description" : "A standard capacity magazine for use with the Browning Automatic Rifle, holds 20 rounds.",
+        "material" : "steel",
+        "volume" : 2,
+        "weight" : 198,
```

Comment:
Clever, but that's not how I determined the weight for this specific item. See https://www.ibiblio.org/hyperwar/USA/ref/FM/PDFs/FM23-15.PDF chapter 1, section 1.

> The weight of the magazine when empty is 7 ounces; when filled, 1 pound 7 ounces.

This was sadly the only instance where I could find reliable data about the actual weight of a magazine. Though I guess bumping it up would be okay if doing so served a gameplay purpose. :V